metaflow/plugins/kfp/kfp.py

```diff
@@ -281,6 +286,16 @@ def _get_retries(node: DAGNode) -> Tuple[int, int]:
 
         return max_user_code_retries, max_user_code_retries + max_error_retries
 
+    @staticmethod
+    def _get_minutes_between_retries(node: DAGNode) -> str:
+        retry_deco = [deco for deco in node.decorators if deco.name == "retry"]
+        if retry_deco:
+            val = retry_deco[0].attributes.get(
+                "minutes_between_retries", BACKOFF_DURATION
```

Comment:
it looks like `minutes_between_retries` has a [default value](https://github.com/zillow/metaflow/blob/feature/kfp/metaflow/plugins/retry_decorator.py#L34) and will always be not None? I assume the BACKOFF_DURATION is not needed.

Reply:
Fixed to use the metaflow decorator default, thanks for catching this!